CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 4.3.0
+
+- Implements Client#close method, allows Infinity for "http2SessionIdleTime" parameter
```

Comment:
What syntax is `Client#close`? Shouldn't this be `Client.close()` ?

Reply:
@faunaee Usually, the `#` sign indicates that this is an instance method, not a static one

Reply:
yes, you are right, I fixed it to `Client.close()`

Reply:
👌 Thanks! Let's keep `Client#close`.

Reply:
ok, sure 👍